### PR TITLE
No issue: Create an asset readme for UI tests

### PR DESCRIPTION
**Pull request by AaronMT**

### CODE REVIEW

**Comment by AaronMT** [comment]
Hm wonder if `fenix/docs` is a better spot

File: app/src/androidTest/java/org/mozilla/fenix/ui/docs/assets.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# README
+_This document is intended to explain specifics in regards to test assets for the Espresso/UI Automator Android UI tests._
+
+# Test application
+* Remote test page managed by Mozilla Mobile Test Engineering at https://github.com/mozilla-mobile/testapp
+* File type download page: https://storage.googleapis.com/mobile_test_assets/test_app/downloads.htm
```
Comment (isabelrios) [inline app/src/androidTest/java/org/mozilla/fenix/ui/docs/assets.md:6]: Nit: missing 'l' at the end, otherwise the page does not load

**Comment by isabelrios** [comment]
In my opinion, and since we have testing docs under `fenix/docs` directory I will add this doc there too. 
But, if possible adding some kind of readme in the test directory, either in `fenix/app/src/androidTest/` or 
`fenix/app/src/androidTest/assets/` would help. Something like this https://github.com/mozilla-mobile/firefox-ios/tree/main/SyncIntegrationTests. 
So that there is info in the tests about the assets easy to find for people testing or using the assets, but also in the docs directory so that that info is shared more broadly.

**Review by sv-ohorvath — COMMENTED** [review]
Do we want the permissions page https://mozilla-mobile.github.io/testapp/permissions documented here as well? And the soon to be merged loginForm page from Andi's PR.

**Comment by AaronMT** [comment]
> In my opinion, and since we have testing docs under `fenix/docs` directory I will add this doc there too.
> But, if possible adding some kind of readme in the test directory, either in `fenix/app/src/androidTest/` or
> `fenix/app/src/androidTest/assets/` would help. Something like this https://github.com/mozilla-mobile/firefox-ios/tree/main/SyncIntegrationTests.
> So that there is info in the tests about the assets easy to find for people testing or using the assets, but also in the docs directory so that that info is shared more broadly.

For this context as these are very specific to ui-tests it makes sense for them to live closer (see Richard's same contextual idea message in Slack) to the location of the tests.

**Comment by AaronMT** [comment]
> Do we want the permissions page https://mozilla-mobile.github.io/testapp/permissions documented here as well? And the soon to be merged loginForm page from Andi's PR. Or is this maybe just for documenting dowloads?

I think those could just be documented in the test themselves, as the context of this document is that there may be re-use for the assets in other tests and one would have to know where to find a list of assets.

File: app/src/androidTest/java/org/mozilla/fenix/ui/docs/assets.md (hunk 1 above)
Comment (AaronMT) [reply app/src/androidTest/java/org/mozilla/fenix/ui/docs/assets.md]: fixed

**Comment by isabelrios** [comment]
> For this context as these are very specific to ui-tests it makes sense for them to live closer (see Richard's same contextual idea message in Slack) to the location of the tests.

Sounds good then.

**Review by isabelrios — APPROVED** [review]

**Pull request merged**